WashU-IT-RIS/coldfront-wustl-fork
# Consolidated repo

ITDEV-34983
https://washu.atlassian.net/browse/ITDEV-34983

To test: 
Follow testing instructions from the readme of this repo and verify tests pass.
Env should be the same as used in `coldfront_plugin_qumulo`, but with and addition variable: `PLUGIN_QUMULO=True`

You can test application by following the instructions here:
https://github.com/WashU-IT-RIS/coldfront/pull/33
